ci-operator/step-registry/hypershift/kubevirt/baremetalds/proxy/hypershift-kubevirt-baremetalds-proxy-commands.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+source "${SHARED_DIR}/packet-conf.sh" && scp "${SSHOPTS[@]}" "${SHARED_DIR}/nested_kubeconfig" "root@${IP}:nested_kubeconfig"
+
+# shellcheck disable=SC2087
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF |& sed -e 's/.*auths.*/*** PULL_SECRET ***/g'
+API_SERVER=\$(cat nested_kubeconfig | yq ".clusters[0].cluster.server" | grep -o '[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}')
+sed -i "1 s|\$| \$API_SERVER|" \$HOME/squid.conf
+
+sudo setenforce 0
+sudo podman stop -t 120 external-squid
+sudo podman run -d --rm \
+     --net host \
+     --volume \$HOME/squid.conf:/etc/squid/squid.conf \
+     --name external-squid \
+     --dns 127.0.0.1 \
+     quay.io/openshifttest/squid-proxy:multiarch
+EOF
+
+echo "Adding proxy-url in kubeconfig"
+sed -i "/- cluster/ a\    proxy-url: http://$IP:8213/" "${SHARED_DIR}"/nested_kubeconfig
```

Comment:
can we just add this and the corresponding health check inline to the `hypershift kubevirt create chain`?

Reply:
nevermind, that's impractical at the moment due to the image not having ssh